new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -0,0 +1,876 @@
+
+    String configServerPodName = domainUid + "-standalone-managed";
+
+    // Make sure that configured managed server is ready 
+    checkPodReadyAndServiceExists(configServerPodName, 
+            domainUid, domainNamespace);
+    logger.info("Configured managed server is RUNNING");
+
+    patchServerStartPolicy(
+         "/spec/managedServers/0/serverStartPolicy", "NEVER");
+    logger.info("Domain is patched to shutdown standalone managed server");
+
+    checkPodDeleted(configServerPodName, domainUid, domainNamespace);
+    logger.info("Configured managed server shutdown success");
+
+    patchServerStartPolicy(
+         "/spec/managedServers/0/serverStartPolicy", "ALWAYS");
+    logger.info("Domain is patched to start standalone managed server");
+
+    checkPodReadyAndServiceExists(configServerPodName, 
+            domainUid, domainNamespace);
+    logger.info("Configured managed server restart success");
+  }
+
+  /**
+   * Start independent managed server by setting serverStartPolicy to IF_NEEDED.
+   * Stop an independent managed server by patching the resource definition with 
+   *  spec/managedServers/0/serverStartPolicy set to NEVER.
+   * Make sure that ONLY the specified managed server is stopped. 
+   * Restart the independent managed server by patching the resource definition 
+   * with spec/managedServers/0/serverStartPolicy set to IF_NEEDED.
+   * Make sure that the specified managed server is in RUNNING state.
+   */
+
+  // The usecase fails NEVER->IF_NEEDED
+  // https://bug.oraclecorp.com/pls/bug/webbug_print.show?c_rptno=31833260
+  @Disabled
+  @Test
+  @DisplayName("Restart the standalone managed server with serverStartPolicy")
+  public void testStandaloneManagedRestartIfNeeded() {
+
+    String configServerPodName = domainUid + "-standalone-managed";
+
+    // Make sure that configured managed server is ready 
+    checkPodReadyAndServiceExists(configServerPodName, 
+            domainUid, domainNamespace);
+    logger.info("Standalone managed server is RUNNING");
+
+    patchServerStartPolicy(
+         "/spec/managedServers/0/serverStartPolicy", "NEVER");
+    logger.info("Domain is patched to shutdown standalone managed server");
+
+    checkPodDeleted(configServerPodName, domainUid, domainNamespace);
+    logger.info("Standalone managed server shutdown success");
+
+    patchServerStartPolicy(
+         "/spec/managedServers/0/serverStartPolicy", "IF_NEEDED");
+    logger.info("Domain is patched to start standalone managed server");
+
+    checkPodReadyAndServiceExists(configServerPodName, 
+            domainUid, domainNamespace);
+    logger.info("Standalone managed server restart success");
+  }
+
+  // This method is needed in this test class, since the cleanup util
+  // won't clean up the images.
+  @AfterAll
+  void tearDown() {
+    // Delete domain custom resource
+    logger.info("Delete domain custom resource in namespace {0}", domainNamespace);
```

Comment:
this is not needed, test infra deletes the domain custom resource as part of cleanup logic

Reply:
removed tearDown()) method